index.html

```diff
@@ -259,6 +259,33 @@ <h3>let's connect</h3>
             </li>
         </ul>
       </section>
+
+      <section id="contact">
+        <div class="contact-bg-form">
+          <div class="contact-form">
+            <h1>Contact me</h1>
+            <p>
+              If you have an application you are 
+              interested in developing, a feature that
+               you need built or a project that needs
+                coding. I’d love to help with it!
+            </p>
+            <form action="/" method="post">
+              <div class="inputs">
+                <input type="text" name="firstname" placeholder="Abdullah Ahmadi">
+              </div>
+              <div class="inputs">
+                <input type="email" name="email" placeholder="abdullahahmadi019@gmail.com">
+              </div>
+              <div class="inputs-msg">
+                <textarea name="msg" id="msg" placeholder="Write your message here..."></textarea>
+              </div>
+              <button type="button" class="contact-btn">Get in touch</button>
+            </form>
```

Comment:
- [ ] Kindly consider setting the `type` attribute of your `button` element to `submit` instead of `button`. This change is essential for ensuring proper form behavior and accessibility. By using `type="submit"`, the button will act as a form submit button, allowing users to submit the form when it's clicked.
- [ ] Using the appropriate `type` attribute not only improves the semantic structure of the HTML but also ensures that the form functions correctly, enhancing the user experience. Therefore, it's best practice to set `type="submit"` for your `button` element when it's intended to submit a form.